### PR TITLE
Feature/theme

### PR DESCRIPTION
**Pull request by khadro**

### CODE REVIEW

File: themes/presto_theme/presto_theme.info.yml [hunk 1]
```diff
@@ -7,11 +7,22 @@ core: 8.x
...
+  sidebar_first: 'Primary'
+  sidebar_second: 'Secondary'
+  footer: 'Footer'
+  page_top: 'Page top'
+  page_bottom: 'Page bottom'
 
-libraries-override:
-  bootstrap/tooltip: false
-  bootstrap/popover: false
+#libraries-override:
```
Comment (jrhodes-sitback) [inline themes/presto_theme/presto_theme.info.yml]: Remove these.

File: themes/presto_theme/gulpfile.babel.js [hunk 2]
```diff
@@ -48,7 +54,7 @@ gulp.task('development', ['js', 'scss'], () => {
 
   gulp.watch(src.scss, ['scss']);
   gulp.watch(src.js, ['js']);
-  // gulp.watch(`${base}/**/*.php`, browserSync.reload);
+  // gulp.watch('**/*.php', browserSync.reload);
```
Comment (jrhodes-sitback) [inline themes/presto_theme/gulpfile.babel.js]: Remove this.

File: themes/presto_theme/scss/modules/_commerce.scss [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+    -webkit-transform: rotate(359deg);
+    transform: rotate(359deg);
+  }
+}
+
+.cart-empty-page {
+  @extend %container;
+}
+
+/* .block-views-blockpresto-product-listing-listing-block {
```
Comment (jrhodes-sitback) [inline themes/presto_theme/scss/modules/_commerce.scss]: Remove if we don't need this.

File: themes/presto_theme/scss/modules/commerce/_shop-listing.scss [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+            }
+          }
+        }
+
+        .add-to-cart-wrapper {
+          left: 0;
+          transform: rotate(90deg);
+          transform-origin: top left;
+
+          /* a {
```
Comment (jrhodes-sitback) [inline themes/presto_theme/scss/modules/commerce/_shop-listing.scss]: Remove if not used.

File: themes/presto_theme/scss/modules/commerce/_shop-listing.scss [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+
+            &::before {
```
Comment (jrhodes-sitback) [inline themes/presto_theme/scss/modules/commerce/_shop-listing.scss]: Remove if not used.

File: themes/presto_theme/scss/modules/commerce/_shop-listing.scss [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+       *///          top: -100%;
+        }
+
+        .view-details-wrapper {
+         // top: 100%;
+          left: 50%;
+          transform: rotate(90deg);
+          transform-origin: bottom left;
+
+          /* a {
```
Comment (jrhodes-sitback) [inline themes/presto_theme/scss/modules/commerce/_shop-listing.scss]: Remove if not used.

File: themes/presto_theme/presto_theme.info.yml (hunk 1 above)
Comment (khadro) [inline themes/presto_theme/presto_theme.info.yml]: done

File: themes/presto_theme/gulpfile.babel.js (hunk 2 above)
Comment (khadro) [inline themes/presto_theme/gulpfile.babel.js]: done

File: themes/presto_theme/scss/modules/_commerce.scss (hunk 3 above)
Comment (khadro) [inline themes/presto_theme/scss/modules/_commerce.scss]: I think this was a leftover from our custom product template. Which is gone. So removed.

File: themes/presto_theme/scss/modules/commerce/_shop-listing.scss (hunk 4 above)
Comment (khadro) [inline themes/presto_theme/scss/modules/commerce/_shop-listing.scss]: All shop listing commented out CSS was for a theme alternative for the animation. Removed for now.

**Pull request merged**